src/scripts/Merge-Workbook.ps1

```diff
@@ -0,0 +1,37 @@
+# Run from root of repo
+
+$workbookPath = "./src/workbooks/optimization"
+$workbookTemplate = "./src/workbooks/optimization/workbook_template.json"

```

Comment:
Instead of creating a copy of the workbook, can we just use the main workbook.json? I'd rather not store the output in our repo, but generate it in the release folder at build-time. This is where merging with Build-Workbook.ps1 might be better.

Reply:
are you referencing the template file ? We are using the workbook.json , the template is just the Overview page with placeholders to inject the rest of the subtemplates

Reply:
I'd rather not have `workbook_tempalte.json` at all. Just use `workbook.json`. We don't need both. It's fine that it pulls in other templates.